godot-core/src/builtin/vector2.rs

```diff
@@ -4,80 +4,113 @@
  * file, You can obtain one at https://mozilla.org/MPL/2.0/.
  */
 
+use std::fmt;
+
 use godot_ffi as sys;
 use sys::{ffi_methods, GodotFfi};
 
-type Inner = glam::f32::Vec2;
-//type Inner = glam::f64::DVec2;
+use crate::builtin::Vector2i;
 
-#[derive(Default, Copy, Clone, Debug, PartialEq)]
+/// Vector used for 2D math using floating point coordinates.
+///
+/// # Description
+///
```

Comment:
I don't think we need an explicit _Description_ title, this just adds vertical space without information. It's implied by Rustdoc that everything past the first line spacing is the "body" of the documentation.

This applies to several other docs in this PR.

Reply:
Copied from official docs. Not a good idea indeed.